contracts/extensions/VotingReputation.sol

```diff
@@ -797,10 +799,10 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTrans
         return MotionState.Staking;
       // If not, did the YAY side stake?
       } else if (motion.stakes[YAY] == requiredStake) {
-        return MotionState.Finalizable;
+        return getSig(motion.action) == NO_ACTION ? MotionState.Finalized : MotionState.Finalizable;
```

Comment:
Stylistically I would do the following:

```
MotionState finalizableState = getSig(motion.action) == NO_ACTION : MotionState.Finalized : MotionState.Finalizable;
```

And then return `finalizableState` as needed. I think `finalizableState` is an appropriate name, since it's `Finalizable` normally but `Finalized` under special circumstances.

Reply:
Another option is to simply `return` in `finalizeMotion` after setting `motion.finalized` to true if the action is the null action, although I suppose you'd want to avoid having users make that transaction.